src/main/java/frc/robot/RobotContainer.java

```diff
@@ -89,13 +106,21 @@ private void configureButtonBindings() {
       OI.Driver.getOuttakeButton().whileTrue(new StartEndCommand(intakeSubsystem::push, intakeSubsystem::stop, intakeSubsystem));
     }
 
+    if (Robot.armEnabled) {
+      OI.Driver.getArmHighButton().onTrue(new ProxyCommand(() -> armSubsystem.transitionToStateTeleop(ArmState.HIGHSCORE)));
+      OI.Driver.getArmMidButton().onTrue(new ProxyCommand(() -> armSubsystem.transitionToStateTeleop(ArmState.MIDSCORE)));
+      OI.Driver.getArmIntakeButton().onTrue(new ProxyCommand(() -> armSubsystem.transitionToStateTeleop(ArmState.CONEINTAKE)));
+    }
+
     if (Robot.buddyBalanceEnabled) {
       OI.Operator.getDownBuddyBalanceButton().and(OI.Operator.getActivateBuddyBalanceButton()).onTrue(new InstantCommand(buddyBalanceSubsystem::deployBuddyBalance, buddyBalanceSubsystem).unless(() -> buddyBalanceSubsystem.getIsDeployed()));
       OI.Operator.getDownBuddyBalanceButton().and(OI.Operator.getActivateBuddyBalanceButton()).onTrue(new InstantCommand(buddyBalanceSubsystem::releaseBuddy, buddyBalanceSubsystem).unless(() -> !buddyBalanceSubsystem.getIsDeployed()));
       OI.Operator.getUpBuddyBalanceButton().and(OI.Operator.getActivateBuddyBalanceButton()).onTrue(new InstantCommand(buddyBalanceSubsystem::retrieveBuddy, buddyBalanceSubsystem).unless(() -> !buddyBalanceSubsystem.getIsDeployed()));
     }
 
-    OI.putControllerButtons();
+    if (!Robot.isCompetition) {
+      OI.putControllerButtons();
+    }
```

Comment:
This is intended to be run in competition and not.

Reply:
We should not run this during competition